### PR TITLE
arc: add page

### PR DESCRIPTION
**Pull request by einverne**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/common/arc.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# arc
+
+> Arcanist(Arc) is a command-line interface to Phabricator.
```
Comment (sbrl) [inline pages/common/arc.md]: Perhaps this will read better?
```suggestion
> Arcanist: A CLI for Phabricator.
```

File: pages/common/arc.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+> More information <https://secure.phabricator.com/book/phabricator/article/arcanist/>.
+
+- Send your code to Differential for review:
```
Comment (sbrl) [inline pages/common/arc.md]: What code is sent? Generally we prefer to use the 3rd person and avoid `your`, `I`, etc. in tldr-pages.

Comment (einverne) [reply pages/common/arc.md]: Sorry, it is the word used on the arc site. How about `Send changes to Differential for review` ?

File: pages/common/arc.md (hunk 1 above)
Comment (einverne) [reply pages/common/arc.md]: Yes, it is much clear.

File: pages/common/arc.md (hunk 2 above)
Comment (sbrl) [reply pages/common/arc.md]: Yeah, that would work @einverne! Thanks.

Sometimes the project's actual wording takes a bit of tweaking to get it to look good in a tldr page :-)

**Pull request merged**